ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -633,6 +633,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: rosa-nightly-sts
+- as: e2e-osd-ccs-aws
+  cron: 0 8 * * 3
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      E2E_RUN_TAGS: '@aws-ipi and @osd_ccs and @network-openshiftsdn and not @fips'
+      OPENSHIFT_VERSION: "4.11"
+      TAG_VERSION: '@4.11'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;OSD_CCS&
```

Comment:
@yasun1 sorry to notice you late. there is change for filter because of hypershift label.
so, it should be 
```yaml
TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;OSD_CCS&
```

`~HyperShiftMGMT&` is added.

Reply:
could merge this PR firstly. and then we could change filter later.

Reply:
Yes, let us merge it first, then I update the filter for rosa and osd. I have applied for the PR review in the testplatform channel. Hopefully, it can be merged ASAP. https://coreos.slack.com/archives/CBN38N3MW/p1665645317462379

Reply:
OK, thanks